app/templates/components/nypr-svg/play-button.hbs

```diff
@@ -0,0 +1,8 @@
+<svg class={{className}} width="22px" height="22px" viewBox="0 0 22 22" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Looks like don't need this file anymore

Reply:
ah, yes, that is for the big play button we are not going to use right now.

Reply:
deleted the file